profiles/features/wd40/package.mask

```diff
@@ -4,6 +4,7 @@
 # Various packages requiring Rust.
 app-admin/ansible
 app-admin/ansible-core
+app-antivirus/clamav
```

Comment:
mask >= 0.105.0 instead, although indeed it's going to be a problem when LTS is discontinued :(